cdap-ui/app/cdap/components/FieldLevelLineage/Fields/Fields.scss

```diff
@@ -19,14 +19,17 @@
 $box-border-color: $green-01;
 $header-border-color: $grey-03;
 $count-font-color: $grey-04;
-$border-color: $grey-05;
 
 .fields-list-container {
+  position: relative;
+
   .fields-box {
     width: 250px;
     border: 2px solid $box-border-color;
     background-color: white;
-    margin: 0 auto;
+    position: absolute;
```

Comment:
Is there any specific reason why we chose absolute positioning rather than use `flexbox` to position the containers?